server/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dmgt-tech/the-usher-server",
-  "version": "1.6.0",
+  "version": "1.6.1",
   "description": "The Usher Authorization Server",
   "engines": {
     "node": ">=14"
```

Comment:
```suggestion
    "node": ">=16"
```

Reply:
@k1ch  even if we upgrade to 18 now, I think I should leave the min. version as 16 as this project still works on 16.